### DISCUSSION TITLE
Funding sources

### DISCUSSION THREAD

**Discussion by zackbatist** [sslarch/sslarch.github.io#52] [General]
During our most recent meeting (September 4, 2024) @SCSchmidt suggested that we compile a list of funding sources that could be leveraged to support our work or the work of our members. We can use this discussion to assemble a relatively stable list of resources, which can then me edited and compiled on the website.

To get us started, here are a few initiatives that seem relevant to our interests. Feel free to reply with additional resources or to input additional info about resources already mentioned.

- [Digital Infrastructure Insights Fund](https://infrastructureinsights.fund/)
  - The D//F (Digital Infrastructure Insights Fund) is a multi-funder initiative by Ford Foundation, Alfred P. Sloan Foundation, Omidyar Network, Schmidt Futures and Open Collective sustaining a platform for researchers and practitioners to better understand how open digital Infrastructure is built and deployed. We’re creating a body of research and implementation insights that advance our goal to ensure a public commons of technology, sustainably developed and maintained, for the benefit of everyone.

- [Software Sustainability Institute Fellowship](https://www.software.ac.uk/programmes/fellowship-programme)
  - The main goals of the Programme are to improve and promote good computational practice across all research disciplines and support those who are doing this important work. We encourage our Fellows to be ambassadors of good practices in their own domains, fields, and/or areas of work.

- [UK Research and Innovation](https://www.ukri.org/what-we-do/creating-world-class-research-and-innovation-infrastructure/digital-research-infrastructure/)

**Comment by zackbatist** [comment]
Some of these are from my own notes, which overlap with a similar list maintained here: https://codeberg.org/teaserbot-labs/delightful-funding

- [Netherlands eScience Center](https://www.esciencecenter.nl/fellowship-programme/)
Aimed at members of the academic research community, working in the Netherlands, who are passionate to act as ambassadors for the use (and reuse) of research software.

- [NLNet Foundation](https://nlnet.nl/)
Offers a variety of grants (including those to foster cross-Atlantic collaboration) geared toward developing key technical infrastructure for broad-level applications  (i.e. jitsi, CryptPad, etc)

- [Prototype Fund](https://nlnet.nl/)
"Low-threshold funding program for free developers in Germany who develop innovative open source software from society and for society." Also: "After approval, you can expect coaching on UX/UI design or project management and optionally on other topics during the funding period"

- [SIDN](https://www.sidnfonds.nl/)
Support initiatives focused on themes including cybersecurity, data autonomy, disinformation and digital accessibility & inclusion. Focused specifically on the Netherlands, no funding for internationally-oriented projects (whatever that means on the internet?) :(

- [Sovereign Tech Fund](https://www.sovereigntechfund.de/)
Offers general investments, bug bounties and fellowhip programs for maintainers of digital infrastructure. Unclear what they mean by digital infrastructure, but seems to imply widely-used open source initiatives (perhaps inspired by the [left-pad incident](https://en.wikipedia.org/wiki/Npm_left-pad_incident)?)

**Comment by zackbatist** [comment]
- [Society of Research Software Engineering (SocRSE) Events and Initiatives Grant](https://society-rse.org/policy-for-socrse-events-and-initiatives-grant/)
The RSE society is able to provide financial support for events and initiatives which advance education particularly but not exclusively amongst research software engineers and/or promote and advance research particularly but not exclusively in all aspects of research that relies on software engineering and to publish the useful results. The society offers financial support of generally up to £1000 per event or initiative. Formal requests for funding can be made via the [Request for Funding Google form](https://forms.gle/imGDC9BVseNP1VR59).

- Some additional strategies and insights on finding funding for smaller-scale open science initiatives, by fellow archaeologist @estherplomp: https://openworking.wordpress.com/2024/11/25/getting-funding-for-small-open-science-initiatives/

**Reply by EstherPlomp** [reply]
Happy to help anyone interested applying to the Fellowship programmes of the eScienceCenter (NL only) or the SSI (international!) - application processes aren't that complicated so can recommend trying!

**Comment by zackbatist** [comment]
Copying a bunch of resources from [Heidi Siebold's recent blog post](https://heidiseibold.kit.com/posts/how-to-get-funding-by-caring-about-research-software-i):

​- [Themed Week ‘Digital Competences in Science’](https://www.volkswagenstiftung.de/de/foerderung/foerderangebot/themenwoche-digitale-kompetenzen-der-wissenschaft-beendet) - Volkswagenstiftung
- ​[Scientific Software](https://klaus-tschira-stiftung.de/foerderungen/naturwissenschaftliche-software/) – Klaus Tschira Stiftung
- [“Research Software Infrastructures” funding programme](https://www.dfg.de/en/research-funding/funding-opportunities/programmes/infrastructure/lis/funding-opportunities/research-software-infrastructures) - DFG
- ​[Foundational Open Source Software Tools Essential to Biomedicine](https://chanzuckerberg.com/newsroom/czi-awards-16-million-for-foundational-open-source-software-tools-essential-to-biomedicine/) - Chan Zuckerberg Initiative, CZI
- ​[Calls from the eScience Center](https://www.esciencecenter.nl/calls-for-proposals/) are generally about code/software - eScience Center Netherlands
- ​[Institutional Support for Open Source Software in Research](https://sloan.org/programs/digital-technology/ospo-loi). Focuses on Open Source Program Offices (OSPOs) - Alfred P. Sloan Foundation
- ​[Platforms Program](https://ardc.edu.au/program/platforms-program/) - Australian Research Data Commons, ARDC
- ​[Software re-use in research](https://www.canarie.ca/to-spur-software-re-use-in-research-canarie-awards-up-to-3-4m-to-research-teams-to-evolve-their-platforms-for-use-by-other-researchers/) - Canadian Network for the Advancement of Research, Industry and Education, CANARIE
- ​[Essential Open Source Software for Science](https://chanzuckerberg.com/eoss/) - Chan Zuckerberg Initiative
- ​[Open & Reusable Research Data & Software](https://www.chistera.eu/call-ord-announcement) - CHIST-ERA
- ​[Open Science Fund](https://www.nwo.nl/en/researchprogrammes/open-science/open-science-fund) - Dutch Research Council, NWO
- ​[FAIR-IMPACT Route 2 support](https://fair-impact.eu/2nd-open-call-route-2-support) includes support actions focused on assessment and improvement of research software - EOSC FAIR-IMPACT
- ​[Call for Proposals to Increase the Usability of Existing Research Software](https://www.dfg.de/foerderung/info_wissenschaft/2022/info_wissenschaft_22_85/index.html) - DFG
- ​[High Priority Open-Source Science](https://nspires.nasaprs.com/external/solicitations/summary.do?solId=%7BB364DBB8-390B-744D-013F-8F4C304B9A63) - NASA
- ​[Support for Open-Source Tools, Frameworks, and Libraries (OSTFL)](https://nspires.nasaprs.com/external/solicitations/summary.do?solId=%7b910CC61E-4616-9958-C26F-F8D9BC5AB8D9%7d&path=&method=init) - NASA
- ​[Cyberinfrastructure for Sustained Scientific Innovation (CSS)](https://www.nsf.gov/publications/pub_summ.jsp?ods_key=nsf21617) - US National Science Foundation, NSF
- ​[Call for Sustainable Software](https://www.esciencecenter.nl/wp-content/uploads/2023/01/SS-2023-Sustainable-software-Call-2023.pdf) - Netherlands eScience Center
- ​[Enhancing biomedical and health-related data and digital platform resources](https://www.ukri.org/opportunity/enhancing-biomedical-and-health-related-data-and-digital-platform-resources/) - UKRI

People are already drawing attention to additional sources [in the replies](https://fosstodon.org/@HeidiSeibold/113842917258776528):

- [R Consortium Infrastructure Steering Committee](https://r-consortium.org/all-projects/)

**Comment by zackbatist** [comment]
The US Research Software Sustainability Institute (URSSI) invites new applications for an Early-Career Fellowship program.

This fellowship offers funding support between $10,000 and $25,000 for research in one of the following areas: AI/ML Integration in Scientific Software Development, Scientific Software Sustainability, or Software Education Research.

The fellowship is open to PhD students, postdoctoral researchers, research software engineers, and research scientists who are less than three years removed from their final degree or appointment.

Applications are due by 25 July 2025 at 11:59 PM PST.

https://urssi.us/blog/2025/07/02/call-for-proposals-urssi-early-career-fellows-round-2/
